### ISSUE TITLE
[RESEARCH] Full audit needed of a11y on mobile view

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#8647]
A couple of mobile a11y issues were brought up while testing category filtering, such as: 

* Tabbing a program card reads out the filtering tags before the program title, which may be confusing
* Focus sometimes jumps from a filter to a random spot on the page 

These issues are more mobile / a11y issues than specific to filtering. We should do a full audit on what our accessibility is like on mobile, and where all our gaps are

**Comment by melanie-exygy** [comment]
@shreyachatterjee00, "Tabbing a program card reads out the filtering tags before the program title, which may be confusing" isn't limited to mobile or limited to program filtering.  I did a bit of research yesterday, looking at the USWDS cards (which we'll be using for the North Star) and the screen reader behavior there is to read the card heading (aka program name) first, then it goes back up to read the image alt text and would presumably keep going and read categories 3rd.  I added a note to check screen reader behavior for this on the North Star program card issue:  #8157.  I think we could remove that bullet point from this program filtering issue.
